samples/deployment/test-network/README.md

```diff
@@ -189,6 +189,19 @@ cd $FPC_PATH/samples/deployment/test-network/fabric-samples/test-network
 ./network.sh down
 ```
 
+## Changing chaincode used by test-network sample
```

Comment:
```suggestion
## FPC HelloWorld on the test-network
```

Reply:
Changed title but not to suggested ... please review @mbrandenburger